### ISSUE TITLE
can mxnet support multiple optimizer scenario

### ISSUE THREAD

**Issue by liumilan** [apache/incubator-mxnet#5838]
i'd like to do some tests
data slice 2 parts
first part used optimizer1 such as adagrad
second part used optimizer2 such as adadelta
and then using the same loss fuction
can mxnet support this ?

**Comment by bhokaal2k** [comment]
Use two separate modules with fixed random seeds to train two different models starting with the exact same weights, same training data sequence(obtained by switching off shuffling in dataIter or possibly by seeding with the same seed), same number of epochs and then compare the two, if it is the comparison you are after?

**Comment by liumilan** [comment]
not yet,i'd like to use one model tranning by 2 optimizers in my scenario Can it possible to support it without changing codes?
@bhokaal2k

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
